HW6/KaziutaRostyslav/hw6.py

```diff
@@ -0,0 +1,28 @@
+# even = []
+# odd = []
+# another = []
+#
+# def numbers(num):
+#     for i in range(num):
+#
+#         if i % 2 == 0:
+#             even.append(i)
+#
+#         if i % 3 == 0:
```

Comment:
it is necessary to replace this condition with this one
if i % 3 == 0 and i % 2 == 1: